ios/RPCModule.swift

```diff
@@ -297,9 +267,9 @@ class RPCModule: NSObject {
   func restoreExistingWalletBackup(_ resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
     do {
       let backupData = try self.readWalletBackup()
```

Comment:
is there a wallet backup?

Reply:
Yes, you can see in the side menu an option called `Restore Las backup wallet` or something similar. When you change to another wallet the App do a backup of your last wallet, this is helpful to switch between two wallets, restoring the backup, and restoring again...